csharp/Svix/Webhook.cs

```diff
@@ -0,0 +1,104 @@
+﻿using System;
+using System.Text;
+using System.Net;
+using System.Security.Cryptography;
+
+using Svix.Exceptions;
+namespace Svix
+{
+    public class Webhook
+    {
+
+        internal static readonly UTF8Encoding SafeUTF8Encoding = new UTF8Encoding(false, true);
+
+        private static readonly int TOLERANCE_IN_SECONDS = 60 * 5;
+        private static string prefix = "whsec_";
+        private byte[] key;
+        public Webhook(string key)
+        {
+            if (key.StartsWith(prefix))
+            {
+                key = key.Substring(prefix.Length);
+            }
+
+            this.key = Convert.FromBase64String(key); ;
+        }
+
+        public void Verify(string payload, WebHeaderCollection headers)
+        {
+            string msgId = headers.Get("svix-id");

```

Comment:
nit: Move keys to const and also use in the tests

Reply:
I'll make them consts, the only question is do we want to expose these as part of the public API? The tests are in their own project so there's no way to access them there without publicly exposing afaik.

Reply:
I made them consts but kept them private, if you think we should expose this we discuss and add them in a separate PR.